.cirrus.yml

```diff
@@ -148,9 +148,9 @@ task:
   get_code_script:
     - git clone https://github.com/PcapPlusPlus/pcapplusplus.github.io.git
   configure_script:
-    - ./configure-linux.sh --default
+    - cmake -DPCAPPP_BUILD_EXAMPLES=OFF -DPCAPPP_BUILD_TESTS=OFF -B build -S .
   build_script:
-    - make libs
+    - cmake --build build -j
```

Comment:
on cirrus -j could trig out of memory. I usually use -j2

Reply:
Fixed in https://github.com/seladb/PcapPlusPlus/pull/1043/commits/78d91c6cd0a570068b72a3904d688b6182033705